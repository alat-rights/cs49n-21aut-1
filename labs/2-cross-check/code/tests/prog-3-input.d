prog-3-input.o: prog-3-input.c ../rpi.h
