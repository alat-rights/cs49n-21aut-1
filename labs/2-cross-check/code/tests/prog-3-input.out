TRACE: calling pi code
TRACE: GET32(0x20200008) = 0x66334873
TRACE: PUT32(0x20200008) = 0x66334871
TRACE: GET32(0x20200008) = 0x66334871
TRACE: PUT32(0x20200008) = 0x66334841
TRACE: GET32(0x20200034) = 0x625558ec
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x238e1f29
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x46e87ccd
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x3d1b58ba
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x507ed7ab
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x2eb141f2
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x41b71efb
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x79e2a9e3
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x7545e146
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x515f007c
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x5bd062c2
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x12200854
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x4db127f8
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x216231b
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x1f16e9e8
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x1190cde7
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x66ef438d
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x140e0f76
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x3352255a
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x109cf92e
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0xded7263
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x7fdcc233
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x1befd79f
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x41a7c4c9
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x6b68079a
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x4e6afb66
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x25e45d32
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x519b500d
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x431bd7b7
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x3f2dba31
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x7c83e458
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x257130a3
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x62bbd95a
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x436c6125
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x628c895d
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x333ab105
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x721da317
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x2443a858
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x2d1d5ae9
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x6763845e
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x75a2a8d4
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x8edbdab
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x79838cb2
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x4353d0cd
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0xb03e0c6
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x189a769b
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x54e49eb4
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x71f32454
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x2ca88611
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x836c40e
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x2901d82
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x3a95f874
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x8138641
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x1e7ff521
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x7c3dbd3d
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x737b8ddc
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x6ceaf087
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x22221a70
TRACE: PUT32(0x2020001c) = 0x100000
TRACE: GET32(0x20200034) = 0x4516dde9
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x3006c83e
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x614fd4a1
TRACE: PUT32(0x20200028) = 0x100000
TRACE: GET32(0x20200034) = 0x419ac241
TRACE: PUT32(0x20200028) = 0x100000
TRACE: exiting after 129 trace statements
TRACE: pi exited cleanly: 69 calls to random
